contrib/99designs/gqlgen/go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/secure-systems-lab/go-securesystemslib v0.9.0 // indirect
 	github.com/shirou/gopsutil/v4 v4.25.8-0.20250809033336-ffcdc2b7662f // indirect
 	github.com/sosodev/duration v1.3.1 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
Where is this dependency coming from?

Reply:
Okay this is because bringing `telemetrytest` in the `testutils` package:

```
github.com/DataDog/dd-trace-go/v2/internal/telemetry/telemetrytest
github.com/stretchr/testify/mock
github.com/stretchr/objx
```

Reply:
Ok, makes sense! So many dependencies 😆